tests/InputFiles/Cce/CharacteristicExtract.yaml

```diff
@@ -24,73 +29,117 @@ ResourceInfo:
 
 Observers:
   VolumeFileName: "CharacteristicExtractUnusedVolume"
+  # The reduction file is where the CCE output will be written.
+  # Specifically, it will be in a `/SpectreR0257.cce` where the number is the
+  # ExtractionRadius specified below.
```

Comment:
Maybe `/SpectreRXXXX.cce` instead of specifying an actual number? I've been using this form in a lot of the other documentation